### DISCUSSION TITLE
Show button on email input fields

### DISCUSSION THREAD

**Discussion by EliDoves** [anonaddy/anonaddy#563] [Feature Requests]
Is it possible to provide a function like "Show button on email input fields" on a browser extension? I think this is helpful to use. Thanks!

**Comment by EliDoves** [comment]
Like this, it only shows up when I sign up for an account, not when I log in, so you can quickly create an alias by pressing this button. this is very convenient
![image](https://github.com/anonaddy/anonaddy/assets/51412618/eee6460d-f7cf-4916-b225-b75245b387e9)

**Comment by willbrowningme** [comment]
Thanks, I'll try to add this when I next update the browser extension.

**Reply by EliDoves** [reply]
Thank you so much! Am looking forward to!
